skin.json

```diff
@@ -214,6 +214,7 @@
     "right": {"fontFamilyName": "alice", "fontString": "\u0073", "fontStyleClass": "icon icon-right"},
     "learn": {"fontFamilyName": "alice", "fontString": "\u0074", "fontStyleClass": "icon icon-learn"},
     "skip": {"fontFamilyName": "alice", "fontString": "\u0075", "fontStyleClass": "icon icon-skip"},
-    "warning": {"fontFamilyName": "fontawesome", "fontString": "\uf06a", "fontStyleClass": ""}
+    "warning": {"fontFamilyName": "fontawesome", "fontString": "\uf06a", "fontStyleClass": ""},
+    "auto": {"fontFamilyName": "alice", "fontString": "\u0064", "fontStyleClass": "icon icon-auto"}
```

Comment:
Yes, this is the icon for automatic bitrate in the selection dialog, the box with an 'A' in it.